modules/evm/rpc/src/lib.rs

```diff
@@ -241,14 +257,15 @@ where
 			.map_err(|err| internal_err(format!("execution fatal: {:?}", err)))?;
 
 		// Determine the highest possible gas limits
-		let max_gas_limit = MAX_GAS_LIMIT;
-		let mut highest = U256::from(request.gas_limit.unwrap_or(max_gas_limit));
+		let max_gas_limit = request.max_gas_limit;
+		let max_storage_limit = request.max_storage_limit;
+		let mut highest = std::cmp::min(request.gas_limit, max_gas_limit);
```

Comment:
`request.max_gas_limit` and `request.max_storage_limit` fields can be removed.

Reply:
@zjb0807 why? I'm using them. I'm also thinking about exposing them to RPC. It may be useful

Reply:
You're right I don't need `max_storage_limit` but I still need `max_gas_limit`, it's been used below